inspectit-ocelot-documentation/docs/getting-started/docker-examples.md

```diff
@@ -30,6 +30,16 @@ Uses Prometheus Server for metrics gathering and storage, Grafana for Dashboards
 
 ![Demo scenario using Prometheus and Jaeger](assets/demo-landscape-prometheus-jaeger.png)
 
+### Demo #3 - Wavefront and Zipkin
+
+Sends Zipkin traces to Wavefront through a Wavefront proxy running as a Docker container.
```

Comment:
Can you also mentioned that no metriks are collected and transmitted to Wavefront in this demo.